### PR TITLE
Set horizontal swing mode

### PR DESCRIPTION
**Pull request by lenisko**

### CODE REVIEW

File: custom_components/gree/climate.py [hunk 1]
```diff
@@ -67,6 +67,7 @@
 
 FAN_MODES = ['Auto', SPEED_LOW, 'Medium-Low', SPEED_MEDIUM, 'Medium-High', SPEED_HIGH, 'Turbo', 'Quiet']
 SWING_MODES = ['Default', 'Swing in full range', 'Fixed in the upmost position', 'Fixed in the middle-up position', 'Fixed in the middle position', 'Fixed in the middle-low position', 'Fixed in the lowest position', 'Swing in the downmost region', 'Swing in the middle-low region', 'Swing in the middle region', 'Swing in the middle-up region', 'Swing in the upmost region']
+HORIZONTAL_SWING_MODES = ['Default', 'Swing in full range', 'Fixed in leftmost position', 'Fixed in middle-left postion', 'Fixed in middle position', 'Fixed in middle-right postion', 'Fixed in rightmost positi n']
```
Comment (sm4rk0) [inline custom_components/gree/climate.py]: "position" is misspelled three times

**Comment by RobHofmann** [comment]
This is still an interesting PR, but as of today i've got still no clue how this could be integrated into HASS. The problem is the UI. For the vertical swing you can say the device has the "SUPPORT_SWING_MODE" flag. But there is no such thing as SUPPORT_HORIZONTAL_SWING_MODE or whatsoever.

I dont want to get loose dropdowns which arent part of the device, so I think this is still impossible as of today.

File: custom_components/gree/climate.py (hunk 1 above)
Comment (lenisko) [reply custom_components/gree/climate.py]: thank you!

**Comment by lenisko** [comment]
@RobHofmann Indeed front is problem in this case. It's up to you if you'll keep this open or close. Cheers!

**Comment by RobHofmann** [comment]
Im closing this. Lets reopen this when support is being brought to HASS.

**Pull request closed without merging**